vendor/github.com/maxlerebourg/simpleredis/README.md

```diff
@@ -28,4 +28,4 @@ if err != nil {
 
 ## Author
 Max Lerebourg @ [Primadviz.com](https://primadviz.com)
-Mathieu Hanotaux
\ No newline at end of file
+Mathieu Hanotaux
```

Comment:
Your linter modified file inside vendor folder. Not good practice

Reply:
It's a final newline from my vscode setup. I am not using a linter.